scripts/checksum

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+var shell = require('shelljs');
+var helpers = require('./helpers');
+var crypto = require('crypto');
+
+function checksum(filename, assignment, rootChecksum) {
+  if(assignment) {
```

Comment:
I prefer to exit early:

```javascript
function checksum(filename, assignment, rootChecksum) {
  if(!assignment) {
    return
  }
  
  ...
```

It keeps the indentation low (and therefore lowers cognitive complexity) and shows the intent instantly: If there is no assignment, don't do anything.